packages/accounts-password/password_server.js

```diff
@@ -1022,6 +1022,52 @@ Meteor.methods(
     }
   });
 
+
+/**
+ * @summary Asynchronously replace an email address for a user. Use this instead of directly
+ * updating the database. The operation will fail if there is a different user
+ * with an email only differing in case. If the specified user has an existing
+ * email only differing in case however, we replace it.
+ * @locus Server
+ * @param {String} userId The ID of the user to update.
+ * @param {String} oldEmail The email address to replace.
+ * @param {String} newEmail The new email address to use.
+ * @param {Boolean} [verified] Optional - whether the new email address should
```

Comment:
I would like to think a bit about the `true` use case. I'm not entirely sure about it. I think we should always send out a verification email (though if set from admin then it might not be necessary).
But what is missing, even if set to `false` I don't see here a verification email being sent out which I think is a must. There also probably should be sent out an email to the old email that it has been changed (maybe an optional flag). This might necessitate new email templates.

Reply:
I side with Nacho. I don't think adding a verification step is needed. It's best to keep operations atomic and single-purpose. Especially that it goes hand in hand with code quality (doing less as in not also handling email verification in the same method) and also following the same flow with already implemented methods.